modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -5,9 +5,52 @@ endif::[]
 
 All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS server by importing a Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) file, or by importing a TLS context from the secrets manager (advanced use case).
 
-Before you can import the TLS context from the secrets manager, you must add the TLS context. 
+== Before You Begin
```

Comment:
This doc is currently organized under Secrets Manager / Secret Groups on the docs site. Given that 1) the new RTF UI is designed so that most users will not use Secrets Manager, and 2) TLS contexts added in Secrets Manager may be used in other parts of the platform, I don't think we should include any RTF-specific info in this doc.

Reply:
Makes sense. Let's discuss where it best fits in the RTF docs...